### PR TITLE
[Php 8] Enable UnionTypesRector

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thank you :clap:

File: src/functions/node_helper.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ function dump_node($node, int $depth = 2): void
     /**
      * @param Node|Node[] $node
      */
-    function print_node($node): void
+    function print_node(Node | array $node): void
```
Comment (TomasVotruba) [inline src/functions/node_helper.php:38]: I think this broke CI on downgrade:
https://github.com/rectorphp/rector-src/runs/2778581424

It's missed for some readon

image

Comment (samsonasik) [reply src/functions/node_helper.php:38]: resolved at https://github.com/rectorphp/rector-src/pull/185

**Pull request merged**